docs/hardware.rst

```diff
@@ -414,6 +414,38 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
+.. _nuc11_recommendation:
+
+Intel 11th-gen NUC
+~~~~~~~~~~~~~~~~~~
+We have tested and can recommend the `Intel NUC11PAHi3 <https://ark.intel.com/content/www/us/en/ark/products/205033/intel-nuc-11-performance-kit-nuc11pahi3.html>`__.
+It provides two storage options: M.2 SSD storage and a 2.5" secondary storage
+option (SSD or HDD).
+
+The NUC11's AX201 wireless hardware is not removable. Before installation of the
+RAM and storage, we recommend that you disconnect the wireless antennae leads
+from the AX201 component. They're the black wires highlighted in the red box in
+the picture. Cover the free ends with electrical tape after disconnecting them.
+
+|NUC11 leads|
+
+Before the initial OS installation, boot into the BIOS by pressing **F2** at
+startup and adjust the system configuration:
+
+- Under **Advanced ▸ Onboard Devices**, disable all onboard devices
+  other than LAN: HD audio, microphone, Thunderbolt, WLAN, Bluetooth,
+  SD card controller, and enhanced consumer infrared.
+
+- Under **Boot ▸ Secure Boot**, uncheck the **Secure Boot** checkbox.
```

Comment:
@eaon, does your NUC's version of the Intel Visual BIOS expose the **Secure Boot** setting as a checkbox?  I have a dropdown menu, in which the desired setting is `Disabled`.

Reply:
You are correct! Thanks for catching!